database/migrations/2020_01_14_171300_add_request_to_followers.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddRequestToFollowers extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('followers', function (Blueprint $table) {
+            $table->boolean('status')->default(1);
```

Comment:
I'd prefer to stick with `accepted` as boolean, to note that it can only be true/false